zenpy/lib/endpoint.py

```diff
@@ -296,8 +296,14 @@ def __call__(self, score=None, sort_order=None):
 
         if sort_order:
             result += '&sort_order={}'.format(sort_order)
-        return result
 
+        if start_time:
```

Comment:
The other endpoints that accept `start_time/end_time` accept either a string or a `datetime` object. There is an example of this here - https://github.com/facetoe/zenpy/blob/master/zenpy/lib/endpoint.py#L156. It might be worth refactoring that snippet out into a function and using it here also to allow users to pass `datetime` objects as well as Unix timestamps.